src/index.html

```diff
@@ -11,7 +11,140 @@
+        </label>
+        <label class="form-label">Name:
+          <input
+            type="text"
+            name="fist-name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-label">How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-label">Full date of birth:
+          <input
+            type="date"
+            name="birth-date"
+            class="date-input"
+          >
+        </label>
+        <label class="form-label">I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label>E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            multiple
+          >
+        </label>
+        <label class="form-label">Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label>Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            id="no"
+          >
+          <label for="no">No</label>
+        </label>
+        <label class="form-label">What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="form-label">What time do you go to bed?
+          <input
+            type="time"
+            name="bedtime"
+            class="time-input"
+            step="2"
+          >
+        </label>
+        <label class="form-label">What are your favorite brand of cars?
+          <select
+            name="cars"
+            multiple size="4"
+            class="cars-selector"
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        </div>
+        <label class="form-label">How do you rate our work
+          <input
+            class="slider"
+            type="range"
+            name="work-rating"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <label>Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            cols="20"
            rows="2"
          >
          </textarea>
```
```suggestion
          <textarea
            name="comments"
            cols="20"
            rows="2"
          ></textarea>
```
To avoid: 
![image](https://user-images.githubusercontent.com/97547824/201763971-fad68fc6-5a9c-4916-a44f-4a07726c7e7e.png)

Reply:
Oh, thanks. Good to know. I`ll rectify it right away.

Reply:
The only problem, that linter doesn`t like it. Gives a mistake:  140:11  error  Tag’s closing bracket should be on the same level as opening one  (closing-and-opening-on-seme-level)